README.md

```diff
@@ -9,3 +9,14 @@ A repo for testing GitHub actions created by the [Oryx](https://github.com/micro
 | v2-beta | Python - Django | [`Source Code`](sampleApps/python/django-app), [`Workflow File`](.github/workflows/v2beta.deployPythonAppToAzureWebApp.yml), [Runs](https://github.com/MicrosoftOryx/githubactions-samples/actions?query=workflow%3A.github%2Fworkflows%2Fv2beta.deployPythonAppToAzureWebApp.yml), [Live Site](http://v2beta-python-sampleapp.azurewebsites.net/uservoice/) |
 | v2-beta | Node - Hello World | [`Source Code`](sampleApps/node/webfrontend), [`Workflow File`](.github/workflows/v2beta.deployNodeAppToAzureWebApp.yml), [Runs](https://github.com/MicrosoftOryx/githubactions-samples/actions?query=workflow%3A.github%2Fworkflows%2Fv2beta.deployNodeAppToAzureWebApp.yml), [Live Site](http://v2beta-node-sampleapp.azurewebsites.net/) |
 | v2-beta | ASP.NET Core MVC | [`Source Code`](sampleApps/dotNetCore/NetCoreApp31.MvcApp), [`Workflow File`](.github/workflows/v2beta.deployDotNetCoreAppToAzureWebApp.yml), [Runs](https://github.com/MicrosoftOryx/githubactions-samples/actions?query=workflow%3A.github%2Fworkflows%2Fv2beta.deployDotNetCoreAppToAzureWebApp.yml), [Live Site](http://v2beta-dotnetcore-sampleapp.azurewebsites.net/) |
+
+
+#### To deploy and build sample apps on-demand
+The command that used for kicking off a build:
+
+```
+curl -X POST https://api.github.com/repos/MicrosoftOryx/githubactions-samples/dispatches \
+-H 'Accept: application/vnd.github.everest-preview+json' \
+-H 'Authorization: token ${{secrets.REPO_ACCESS_TOKEN}}' \
```

Comment:
`REPO_ACCESS_TOKEN` => `YOUR_PERSONAL_ACCESS_TOKEN` ? this way it will be clear

Reply:
Just to clarify, it has to be the access token of  owner account of githubactions-samples repo. So either users forked this repo to be their own repo and use their personal one or it still needs to be yours.

Reply:
Sure, I get that. I am trying to say is that the word `REPO_ACCESS_TOKEN` might not be entirely clear to the reader.